### ISSUE TITLE
ndctl clear-errors all -s does multiple ARS scrubs

### ISSUE THREAD

**Issue by etsaur4** [pmem/ndctl#104]
The current implementation of ndctl clear-errors takes a very long time, because a full scrub happens for every namespace.

Doing a full system ARS scrub is obviously not necessary, it just needs to happen once

**Comment by etsaur4** [comment]
Doing the below is more takes just a fraction of the time:
ndctl start-scrub
ndctl wait-scrub
ndctl clear-errors all

**Comment by djbw** [comment]
Agree, seems simple enough to only perform the scrub once per-bus in a single "ndctl clear-error -s" invocation.
